proxylib/proxylib/types.go

```diff
@@ -21,12 +21,8 @@ import "C"
 
 // Mirror C types to be able to use them in other Go files and tests.
 
-type OpType uint32
-type OpError uint32
-type Op struct {
-	op      uint32
-	n_bytes uint32 // The number of bytes of data the operation 'op' applies to.
-}
+type OpType int64
```

Comment:
Generally, you can't use `C.` types without using Cgo, and I have wanted to limit Go-files using Cgo tp the minimum. On reason for this is that any types defined if different Go packages, even if defined identically, are not the same type. 

Also, in a library interface it is safer to used explicitly sized types rather than assume both sides agree on a size of a type.